spec/spec_helper.rb

```diff
@@ -68,3 +68,18 @@ def missing_methods
     expected.instance_methods - actual.instance_methods
   end
 end
+
+
+def configure(default_currency: nil, legacy_json_format: nil, legacy_deprecations: nil, legacy_default_currency: nil, parser: nil)
+  old_config = Money.config.dup
```

Comment:
don't think I need the dup here
```suggestion
  old_config = Money.config
```